client/my-sites/plugins-wpcom/plugin-panel.jsx

```diff
@@ -22,8 +22,8 @@ export const PluginPanel = React.createClass( {
 				<Card className="wpcom-plugin-panel__panel-footer" href={ standardPluginsLink }>
 					{ this.translate( 'View all standard plugins' ) }
 				</Card>
-				<PremiumPluginsPanel />
-				<BusinessPluginsPanel />
+				<PremiumPluginsPanel siteSlug={ siteSlug } />
+				<BusinessPluginsPanel siteSlug={ siteSlug } />
```

Comment:
Yes, I think make sense. I'm trying to understand and going forward with this implementation. :+1: